scripts/pythonContractConverter.py

```diff
@@ -0,0 +1,115 @@
+# assumes all return types are strings
+def parseContract(file, funcName):
+    import re
+
+    fin = -1
+    try:
+        fin = open(file)
+    except:
+        print("[ERROR] {fname} cannot be opened".format(fname = file))
+        return ""
+
+    data = fin.read()
+    fin.close()
+    pattern = re.compile(r'#.*?\n')
+    data = re.sub(pattern, '', data)
+    pattern = re.compile(r"'''.*?'''", re.DOTALL)
+    data = re.sub(pattern, '', data)
+    if(("def " + funcName + "(") not in data):
+        print("[ERROR] {fname} does not contain a definition for \"{func}\"".format(fname = file, func = funcName))
+        return ""
+    data = data.replace('\n', '\n|')
+    data_lines = data.split('|')
+
+    idx = 0
+    for i in range(len(data_lines)):
+        if("def " + funcName + "(" in data_lines[i]):
+            idx = i
+            break
+    data_lines = data_lines[idx:]
+
+
+    while(True):
```

Comment:
no need for brackets

```suggestion
    while True:
```